ush/cam/evs_href_prepare.sh

```diff
@@ -115,7 +115,7 @@ if [ "$data" = "ccpa01h03h" ] ; then
    done
    if [ "$SENDMAIL" = "YES" ] && [ "$missing_ccpa" -gt "0" ] ; then
       export subject="CCPA Data Missing for EVS ${COMPONENT}"
-      echo "Warning:  No CCPA data available for ${VDATE}" > mailmsg
+      echo "WARNING:  No CCPA data available for ${VDATE}" > mailmsg
       echo -e "`cat $DATA/job${data}${domain}_missing_ccpa_list`" >> mailmsg
```

Comment:
We need to move this outside the SENDMAIL block, like you did below for RAP prepbufr, including the full path.